packages/babel-plugin-transform-function-sent/test/fixtures/function-sent/after-yield/actual.js

```diff
@@ -0,0 +1,5 @@
+(function* () {
```

Comment:
It seems like for this case, we don't technically need the `_skipFirstGeneratorNext` wrapper, because `function.sent` is never used before the first `yield`. We don't have much in the way of flow analysis right now, but it could be super nice to only inject the wrapper when it is strictly necessary.